util/merkle/src/triedbmut.rs

```diff
@@ -417,6 +417,10 @@ impl<'a> Trie for TrieDBMut<'a> {
 
         t.get(key)
     }
+
+    fn is_complete(&self) -> bool {
+        TrieDB::try_new(self.db, self.root).map(|t| t.is_complete()).unwrap_or(false)
```

Comment:
```rust
TrieDB::try_new(self.db, self.root).map_or(false, |t| t.is_complete())
```

Reply:
[`Result`](https://doc.rust-lang.org/std/result/enum.Result.html#method.ok) doesn't have `map_or` function.

Reply:
https://doc.rust-lang.org/nightly/std/result/enum.Result.html#method.map_or Sorry. It is nightly yet.